reflex/components/component.py

```diff
@@ -279,7 +279,11 @@ def get_triggers(self) -> Set[str]:
         Returns:
             The event triggers.
         """
-        return EVENT_TRIGGERS | set(self.get_controlled_triggers())
+        return (
+            EVENT_TRIGGERS
+            | set(self.get_controlled_triggers())
+            | set((constants.ON_MOUNT, constants.ON_UNMOUNT))
```

Comment:
Shouldn't we just add these to the default `EVENT_TRIGGERS` ?

Reply:
i went back and forth on this, and ultimately did it this way because these aren't "real" event triggers that come from browser events, but are synthetic events that reflex handles.